Description: Don't use Ivy for dependencies resolution
 as not enough debian packages provide Maven/Ivy metadata.
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2011-08-20
Forwarded: not-needed

--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.core/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.core/ivy.xml
@@ -25,9 +25,11 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
 		<dependency org="org.apache.log4j" name="com.springsource.org.apache.log4j" rev="1.2.15" conf="optional, log4j->compile"/>
 		<dependency org="org.aspectj" name="com.springsource.org.aspectj.weaver" rev="${aspectj.version}" conf="optional, aspectj->compile"/>
+-->
 		<!--<dependency org="org.springframework" name="org.springframework.asm" rev="latest.integration" conf="optional->compile"/>-->
 		<!-- test dependencies -->
 		<dependency org="javax.servlet" name="com.springsource.javax.servlet" rev="2.5.0" conf="test->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.web/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.web/ivy.xml
@@ -35,6 +35,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.caucho" name="com.springsource.com.caucho" rev="3.2.1"
                     conf="optional, hessian, burlap->compile"/>
         <dependency org="com.sun.syndication" name="com.springsource.com.sun.syndication" rev="1.0.0"
@@ -66,6 +67,7 @@
                     conf="optional, jackson->compile"/>
         <dependency org="org.jdom" name="com.springsource.org.jdom" rev="1.1.0"
                     conf="optional, feed->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.aop" rev="latest.integration"
                     conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration"
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.aspects/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.aspects/ivy.xml
@@ -25,17 +25,21 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
 		<dependency org="org.aspectj" name="com.springsource.org.aspectj.weaver" rev="${aspectj.version}" conf="optional, aspectj->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="test->compile"/>
 		<dependency org="org.springframework" name="org.springframework.transaction" rev="latest.integration" conf="optional, tx->compile"/>
 		<dependency org="org.springframework" name="org.springframework.orm" rev="latest.integration" conf="optional, orm->compile"/>
 		<dependency org="org.springframework" name="org.springframework.test" rev="latest.integration" conf="test->runtime"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="test->runtime"/>
 		<dependency org="org.springframework" name="org.springframework.context.support" rev="latest.integration" conf="test->runtime"/>
+<!--
 		<dependency org="javax.persistence" name="com.springsource.javax.persistence" rev="1.0.0" conf="optional, orm->compile"/>
 		<dependency org="org.junit" name="com.springsource.org.junit" rev="${junit.version}" conf="test->compile"/>
 		<dependency org="javax.mail" name="com.springsource.javax.mail" rev="1.4.0" conf="optional, test->compile"/>
+-->
 	</dependencies>
 
 </ivy-module>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.jdbc/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.jdbc/ivy.xml
@@ -26,16 +26,20 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.mchange.c3p0" name="com.springsource.com.mchange.v2.c3p0" rev="0.9.1.2" conf="optional, c3p0->compile"/>
 		<dependency org="javax.transaction" name="com.springsource.javax.transaction" rev="1.1.0" conf="provided->runtime"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="optional, jndi->compile"/>
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.transaction" rev="latest.integration" conf="compile->compile"/>
+<!--
 		<dependency org="org.hsqldb" name="com.springsource.org.hsqldb" rev="1.8.0.9" conf="optional->compile"/>
 		<dependency org="com.h2database" name="com.springsource.org.h2" rev="1.0.71" conf="optional->compile"/>
 		<dependency org="org.apache.derby" name="com.springsource.org.apache.derby" rev="10.5.1000001.764942" conf="optional->compile"/>
+-->
 		<!-- test dependencies -->
 		<dependency org="org.junit" name="com.springsource.org.junit" rev="${junit.version}" conf="test->runtime"/>
 		<dependency org="org.easymock" name="com.springsource.org.easymock" rev="2.5.1" conf="test->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.transaction/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.transaction/ivy.xml
@@ -25,12 +25,14 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.ibm.websphere" name="com.springsource.com.ibm.websphere.uow" rev="6.0.2.17" conf="provided, websphere->compile"/>
 		<dependency org="javax.ejb" name="com.springsource.javax.ejb" rev="3.0.0" conf="provided, ejb->compile"/>
 		<dependency org="javax.resource" name="com.springsource.javax.resource" rev="1.5.0" conf="provided, jca->compile"/>
 		<dependency org="javax.transaction" name="com.springsource.javax.transaction" rev="1.1.0" conf="provided->compile"/>
 		<dependency org="org.aopalliance" name="com.springsource.org.aopalliance" rev="1.0.0" conf="compile->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.aop" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="compile->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.expression/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.expression/ivy.xml
@@ -23,7 +23,9 @@
 
 	<dependencies>
 		<!-- compile dependencies -->
+<!--
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->runtime" />
+-->
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="compile->runtime" />
 		<!-- test dependencies -->
 		<dependency org="org.apache.log4j" name="com.springsource.org.apache.log4j" rev="1.2.15" conf="test->runtime"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.beans/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.beans/ivy.xml
@@ -23,10 +23,12 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="javax.el" name="com.springsource.javax.el" rev="1.0.0" conf="provided->compile"/>
 		<dependency org="javax.inject" name="com.springsource.javax.inject" rev="1.0.0" conf="provided->compile"/>
 		<dependency org="net.sourceforge.cglib" name="com.springsource.net.sf.cglib" rev="2.2.0" conf="optional, bytecode-proxy->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="compile->compile"/>
 		<!-- test dependencies -->
 		<dependency org="org.apache.log4j" name="com.springsource.org.apache.log4j" rev="1.2.15" conf="test->runtime"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.web.portlet/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.web.portlet/ivy.xml
@@ -23,6 +23,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="javax.el" name="com.springsource.javax.el" rev="1.0.0" conf="provided->compile"/>
 		<dependency org="javax.portlet" name="com.springsource.javax.portlet" rev="2.0.0" conf="provided->compile"/>
 		<dependency org="javax.servlet" name="com.springsource.javax.servlet" rev="2.5.0" conf="compile->compile"/>
@@ -30,6 +31,7 @@
 		<dependency org="javax.servlet" name="com.springsource.javax.servlet.jsp.jstl" rev="1.1.2" conf="test->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.fileupload" rev="1.2.0" conf="optional, commons-fileupload->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="compile->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.test/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.test/ivy.xml
@@ -22,6 +22,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="javax.activation" name="com.springsource.javax.activation" rev="1.1.0" conf="provided->compile"/>
 		<dependency org="javax.el" name="com.springsource.javax.el" rev="1.0.0" conf="provided->compile"/>
 		<dependency org="javax.inject" name="com.springsource.javax.inject" rev="1.0.0" conf="test->compile"/>
@@ -38,6 +39,7 @@
 		<dependency org="org.junit" name="com.springsource.junit" rev="3.8.2" conf="optional->compile"/>
 		<dependency org="org.junit" name="com.springsource.org.junit" rev="${junit.version}" conf="optional->compile"/>
 		<dependency org="org.slf4j" name="com.springsource.slf4j.jcl" rev="${slf4j.version}" conf="test->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="optional->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="optional->compile"/>
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="optional->compile"/>
@@ -47,7 +49,9 @@
 		<dependency org="org.springframework" name="org.springframework.web" rev="latest.integration" conf="optional->compile"/>
 		<dependency org="org.springframework" name="org.springframework.web.portlet" rev="latest.integration" conf="optional->compile"/>
 		<dependency org="org.springframework" name="org.springframework.web.servlet" rev="latest.integration" conf="optional->compile"/>
+<!--
 		<dependency org="org.testng" name="com.springsource.org.testng" rev="${testng.version}" conf="optional->compile"/>
+-->
 	</dependencies>
 
 </ivy-module>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.spring-library/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.spring-library/ivy.xml
@@ -34,7 +34,7 @@
 		<dependency org="org.springframework" name="org.springframework.web" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.web.servlet" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.web.portlet" rev="latest.integration" conf="compile->compile"/>
-		<dependency org="org.aopalliance" name="com.springsource.org.aopalliance" rev="1.0.0" conf="compile->compile"/>
+		<!--<dependency org="org.aopalliance" name="com.springsource.org.aopalliance" rev="1.0.0" conf="compile->compile"/>-->
 	</dependencies>
 
 </ivy-module>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.integration-tests/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.integration-tests/ivy.xml
@@ -21,6 +21,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.caucho" name="com.springsource.com.caucho" rev="3.2.1" conf="test->compile"/>
 		<dependency org="com.ibm.websphere" name="com.springsource.com.ibm.websphere.uow" rev="6.0.2.17" conf="test->compile"/>
 		<dependency org="com.opensymphony.quartz" name="com.springsource.org.quartz" rev="1.6.2" conf="test->compile"/>
@@ -75,6 +76,7 @@
 		<dependency org="org.hibernate" name="com.springsource.org.hibernate.ejb" rev="3.4.0.GA" conf="test->compile"/>
 		<dependency org="org.hsqldb" name="com.springsource.org.hsqldb" rev="1.8.0.9" conf="test->compile"/>
 		<dependency org="org.jruby" name="com.springsource.org.jruby" rev="1.4.0" conf="optional, runtime->compile"/>
+-->
 <!--		<dependency org="org.springframework" name="org.springframework.asm" rev="latest.integration" conf="test->runtime"/>-->
 		<dependency org="org.springframework" name="org.springframework.aop" rev="latest.integration" conf="test->compile"/>
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="test->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.context/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.context/ivy.xml
@@ -34,6 +34,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="edu.emory.mathcs.backport" name="com.springsource.edu.emory.mathcs.backport" rev="3.0.0" conf="optional, backport-util-concurrent->compile"/>
 		<dependency org="javax.annotation" name="com.springsource.javax.annotation" rev="1.0.0" conf="provided, ejb->compile"/>
 		<dependency org="javax.ejb" name="com.springsource.javax.ejb" rev="3.0.0" conf="provided, ejb->compile"/>
@@ -51,6 +52,7 @@
 		<dependency org="org.hibernate" name="com.springsource.org.hibernate.validator" rev="4.1.0.GA" conf="optional->compile"/>
 		<dependency org="org.joda" name="com.springsource.org.joda.time" rev="1.6.0" conf="optional->compile"/>
 		<dependency org="org.jruby" name="com.springsource.org.jruby" rev="1.4.0" conf="optional, jruby->compile"/>
+-->
 <!--		<dependency org="org.springframework" name="org.springframework.asm" rev="latest.integration" conf="compile->compile"/>-->
 		<dependency org="org.springframework" name="org.springframework.aop" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
@@ -58,7 +60,9 @@
 		<dependency org="org.springframework" name="org.springframework.expression" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.instrument" rev="latest.integration" conf="optional, instrumentation->compile"/>
 		<!-- test dependencies -->
+<!--
 		<dependency org="javax.inject" name="com.springsource.org.atinject.tck" rev="1.0.0" conf="provided->compile"/>
+-->
 		<dependency org="javax.transaction" name="com.springsource.javax.transaction" rev="1.1.0" conf="test->compile"/>
 		<dependency org="javax.xml.rpc" name="com.springsource.javax.xml.rpc" rev="1.1.0" conf="test->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.pool" rev="1.5.3" conf="test->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.oxm/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.oxm/ivy.xml
@@ -28,6 +28,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.thoughtworks.xstream" name="com.springsource.com.thoughtworks.xstream" rev="1.3.1"
 					conf="optional, xstream->compile"/>
 		<dependency org="javax.xml.bind" name="com.springsource.javax.xml.bind" rev="2.1.7"
@@ -36,13 +37,16 @@
 					conf="optional, jibx->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1"
 					conf="compile, commons-logging->compile"/>
+-->
 		<!--Xerces dependency should be removed when BRITS-259 is fixed-->
+<!--
 		<dependency org="org.apache.xerces" name="com.springsource.org.apache.xerces" rev="2.8.1"
 					conf="optional, castor->compile"/>
 		<dependency org="org.apache.xmlbeans" name="com.springsource.org.apache.xmlbeans" rev="2.4.0"
 					conf="optional, xmlbeans->compile"/>
 		<dependency org="org.codehaus.castor" name="com.springsource.org.castor" rev="1.2.0"
 					conf="optional, castor->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration"
 					conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration"
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.web.struts/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.web.struts/ivy.xml
@@ -23,12 +23,14 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="javax.servlet" name="com.springsource.javax.servlet" rev="2.5.0" conf="provided->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1"
                     conf="compile, commons-logging->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.beanutils" rev="1.7.0"
                     conf="compile->compile"/>
 		<dependency org="org.apache.struts" name="com.springsource.org.apache.struts" rev="1.2.9" conf="compile->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration"
                     conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration"
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.orm/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.orm/ivy.xml
@@ -33,6 +33,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.oracle.toplink.essentials" name="com.springsource.oracle.toplink.essentials" rev="2.0.0.b41-beta2" conf="optional, toplink->compile"/>
 		<dependency org="org.aopalliance" name="com.springsource.org.aopalliance" rev="1.0.0" conf="optional, aop->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
@@ -47,6 +48,7 @@
 		<dependency org="javax.persistence" name="com.springsource.javax.persistence" rev="1.0.0" conf="provided, jpa->compile"/>
 		<dependency org="javax.servlet" name="com.springsource.javax.servlet" rev="2.5.0" conf="provided, web->compile"/>
 		<dependency org="javax.transaction" name="com.springsource.javax.transaction" rev="1.1.0" conf="provided->runtime"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.aop" rev="latest.integration" conf="optional, aop->compile"/>
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="optional, jndi->compile"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.instrument.tomcat/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.instrument.tomcat/ivy.xml
@@ -20,8 +20,10 @@
 		<artifact name="notice" type="notice" ext="txt"/>
 	</publications>
 
+<!--
 	<dependencies>
 		<dependency org="org.apache.catalina" name="com.springsource.org.apache.catalina" rev="6.0.16" conf="provided->compile"/>
 	</dependencies>
+-->
 
 </ivy-module>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.jms/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.jms/ivy.xml
@@ -23,6 +23,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="javax.jms" name="com.springsource.javax.jms" rev="1.1.0" conf="provided->compile"/>
 		<dependency org="javax.resource" name="com.springsource.javax.resource" rev="1.5.0"
 					conf="provided, jca->compile"/>
@@ -35,6 +36,7 @@
 					conf="optional, commons-pool->compile"/>
 		<dependency org="org.easymock" name="com.springsource.org.easymock" rev="2.5.1" conf="test->compile"/>
 		<dependency org="org.junit" name="com.springsource.org.junit" rev="${junit.version}" conf="test->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.aop" rev="latest.integration"
 					conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration"
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.web.servlet/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.web.servlet/ivy.xml
@@ -32,6 +32,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.sun.syndication" name="com.springsource.com.sun.syndication" rev="1.0.0"
                     conf="optional, feed->compile"/>
 		<dependency org="com.lowagie.text" name="com.springsource.com.lowagie.text" rev="2.0.8"
@@ -65,6 +66,7 @@
                     conf="optional, velocity->compile"/>
 		<dependency org="org.codehaus.jackson" name="com.springsource.org.codehaus.jackson.mapper" rev="1.4.2"
                     conf="optional, jackson->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration"
                     conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration"
@@ -97,7 +99,7 @@
                     conf="test->compile"/>
 		<dependency org="org.springframework" name="org.springframework.orm" rev="latest.integration"
                     conf="test->compile"/>
-		<dependency org="javax.transaction" name="com.springsource.javax.transaction" rev="1.1.0" conf="provided->runtime"/>
+		<!--<dependency org="javax.transaction" name="com.springsource.javax.transaction" rev="1.1.0" conf="provided->runtime"/>-->
 		<dependency org="javax.validation" name="com.springsource.javax.validation" rev="1.0.0.GA" conf="test->compile"/>
 		<dependency org="org.hibernate" name="com.springsource.org.hibernate.validator" rev="4.1.0.GA" conf="test->runtime"/>
 		<dependency org="org.slf4j" name="com.springsource.slf4j.jcl" rev="${slf4j.version}" conf="test->runtime"/>
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.aop/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.aop/ivy.xml
@@ -26,12 +26,14 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.jamonapi" name="com.springsource.com.jamonapi" rev="2.4.0" conf="optional, jamon->compile"/>
 		<dependency org="net.sourceforge.cglib" name="com.springsource.net.sf.cglib" rev="2.2.0" conf="optional, bytecode-proxy->compile"/>
 		<dependency org="org.aopalliance" name="com.springsource.org.aopalliance" rev="1.0.0" conf="compile->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.pool" rev="1.5.3" conf="optional, pooling->compile"/>
 		<dependency org="org.aspectj" name="com.springsource.org.aspectj.weaver" rev="${aspectj.version}" conf="optional, aspectj->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="compile->compile"/>
 		<!-- test dependencies -->
--- libspring-java-3.0.6.RELEASE.orig/projects/org.springframework.context.support/ivy.xml
+++ libspring-java-3.0.6.RELEASE/projects/org.springframework.context.support/ivy.xml
@@ -31,6 +31,7 @@
 	</publications>
 
 	<dependencies>
+<!--
 		<dependency org="com.bea.commonj" name="com.springsource.commonj" rev="1.1.0" conf="optional, commonj->compile"/>
 		<dependency org="com.opensymphony.quartz" name="com.springsource.org.quartz" rev="1.6.2" conf="optional, quartz->compile"/>
 		<dependency org="javax.activation" name="com.springsource.javax.activation" rev="1.1.0" conf="provided->compile"/>
@@ -41,6 +42,7 @@
 		<dependency org="org.apache.commons" name="com.springsource.org.apache.commons.logging" rev="1.1.1" conf="compile, commons-logging->compile"/>
 		<dependency org="org.apache.velocity" name="com.springsource.org.apache.velocity" rev="1.5.0" conf="optional, velocity->compile"/>
 		<dependency org="org.freemarker" name="com.springsource.freemarker" rev="2.3.15" conf="optional, freemarker->compile"/>
+-->
 		<dependency org="org.springframework" name="org.springframework.beans" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.context" rev="latest.integration" conf="compile->compile"/>
 		<dependency org="org.springframework" name="org.springframework.core" rev="latest.integration" conf="compile->compile"/>
